docs/content/blog/posts/2023/23-01-09-constraining-the-model.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Layered: Constraining the Model"
+menu:
+  main:
+    identifier: "23-01-09-constraining-the-model"
+    parent: "2023"
+    weight: 10
+---
+
+_By Sören Domrös, January 9, 2023_
+
+Since it is often desired to constrain the layout in a specific way to achieve a desired layout or to increase layout stability, this post should summarize all current (ELK 0.8.1) and future (planned for 0.9.0) ways to do that. The following will focus primarily on constraining the node order. If you wish to constrain the ports have a look at the [`portConstraints`](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-portConstraints.html) property and [this](https://rtsys.informatik.uni-kiel.de/elklive/examples.html?e=general%2Fspacing%2Fports) example.
+
+## Layer Constraints
+
+The [`layerConstraint`](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-layering-layerConstraint.html) property allows constraining a node to the first or last layer. This option is quite basic and should be compatible with the Partition and model order cycle breaking. Other approaches might produce unexpected results when used together.
+
+## Interactive Constraints
+
+Interactive layout works by having previously defined positions and by using the following layout options for the layered algorithm:
+```
+cycleBreaking.strategy: INTERACTIVE
+layering.strategy: INTERACTIVE
+crossingMinimization.semiInteractive: true
+separateConnectedComponents: false
+```
+
+There are two different ways to get positions for your nodes.
+
+### Pseudo Positions
```

Comment:
The difference in presentation between `h2` and `h3` elements is marginal: 32px vs. 28px – it looks like a new section rather than a subsection. That makes it harder to understand what you mean with "pseudo positions" and what they are used for.